src/github.com/getlantern/lantern-android/app/src/go/Seq.java

```diff
@@ -36,6 +36,7 @@ public Seq() {
 	public native float readFloat32();
```

Comment:
This file is generated by the build process, we should probably not take it into the source tree as it is not really editable.